languages/julia/exercises/concept/lasagna/.docs/introduction.md

```diff
@@ -1,3 +1,5 @@
+## functions-introduction
+
 The entire Julia track will require you to treat your solution like small libraries, i.e. you need to define functions, types etc. which will then be run against a test suite.
 For that reason, we will introduce named functions as the very first concept.
```

Comment:
@ErikSchierboom This is more of an introduction to the exercise than part of the concept description. Should the header go below that in this case?

Reply:
Probably. It does depend a bit on how @iHiD will parse the markdown. @iHiD could you enlighten us?

Reply:
This content will be what is seen on the LHS [here](http://exercism.lol/tracks/julia/concepts/functions-introduction) before someone has completed the exercise. The `## functions-introduction` will be removed, and probably replaced with `## Introduction` 

e.g.

<img width="794" alt="Screenshot 2020-11-09 at 19 26 01" src="https://user-images.githubusercontent.com/286476/98586979-6b599e80-22c1-11eb-85da-c9bd2aa49ab1.png">

Reply:
(So yes, move the header below that)